winui/Shadow/Getting-Started.md

```diff
@@ -0,0 +1,181 @@
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:GettingStarted"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:syncfusion="using:Syncfusion.UI.Xaml.Core"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Grid>
+       <syncfusion:SfShadow Name="buttonShadow"
+                      Color="#40000000"
+                      BlurRadius="8"
+                      OffsetX="4"
+                      OffsetY="4"
+                      ShadowCornerRadius="5">
+          <Button x:Name="buttonShadow" Height="50" width="100" Content="Button"         
+              CornerRadius="5"/>
+       </syncfusion:SfShadow>
+    </Grid>
+</Page>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.UI.Xaml.Core;
+
+namespace GettingStarted
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+           this.InitializeComponent();
+           // Creating an instance of the Shadow control.
+           SfShadow shadow = new SfShadow();
+
+           //Setting Color, BlurRadius, OffsetX, OffsetY, ShadowCornerRadius of Shadow.
+           shadow.Color = Color.FromArgb(64, 0, 0, 0);
+           shadow.BlurRadius = 8;
+           shadow.OffsetX = 4;
+           shadow.OffsetY = 4;
+           shadow.ShadowCornerRadius = 5;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Shadow added to the Button Control](Shadow_images/Button.png)
+
+
+## Adding Image for Shadow
+
+If you want to add the image for `Shadow`, add the image source in the shadow control. 
+
+{% tabs %}
+{% highlight XAML %}
+
+<core:SfShadow Name="imageShadow"
+               Color="#40000000"
+ 			   BlurRadius="8"
+               OffsetX="4"
+               OffsetY="4">
+    <Image Height="200" Width="250" 
+   Source="/Assets/Shadow/Ellipse_Shadow.png/>
+</core:SfShadow>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.UI.Xaml.Core;
+
+namespace GettingStarted
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+           this.InitializeComponent();
+           // Creating an instance of the Shadow control.
+           SfShadow shadow = new SfShadow();
+
+           //Setting Color, BlurRadius, OffsetX, OffsetY of Shadow.
+           shadow.Color = Color.FromArgb(64, 0, 0, 0);
+           shadow.BlurRadius = 8;
+           shadow.OffsetX = 4;
+           shadow.OffsetY = 4;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+
+![Shadow added to the Image](Shadow_images/Image.png)
```

Comment:
@Priyanka-Vijayasankar Follow image name properly with platform name. for eg: winui_shadow_image.

Reply:
Followed the image name as mentioned above.